### PR TITLE
Elaborate on roadmap and contributing docs

### PR DESCRIPTION
**Pull request by kate-goldenring**
**What this PR does / why we need it**:
Expands docs on contributing by
- Referencing our project roadmap in our roadmap doc
- Referencing roadmap doc from README
- Adding community meetings doc to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,8 +46,12 @@ Try the [end to end demo](./docs/end-to-end-demo.md) of Akri to see Akri discove
 ## Roadmap
 Akri was built to be extensible. We currently have ONVIF, udev, OPC UA Discovery Handlers, but as a community, we hope to continuously support more protocols. We have created a [Discovery Handler implementation roadmap](./docs/roadmap.md#implement-additional-discovery-handlers) in order to prioritize development of Discovery Handlers. If there is a protocol you feel we should prioritize, please [create an issue](https://github.com/deislabs/akri/issues/new/choose), or better yet, contribute the implementation!
```
Comment (edrickwong) [inline README.md:47]: I would say "Akri **is** built to be extensible

File: docs/contributing.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Contributing 
-This project welcomes contributions and suggestions. We are excited for you to join the Akri community!
+Want to work on Akri with us? 🎉 We are actively looking for new contributors along all stages of the contribution journey, from casual contributors to reviewes and maintainers.
```
Comment (edrickwong) [inline docs/contributing.md:2]: Reviewers*

File: README.md (hunk 1 above)
Comment (kate-goldenring) [reply README.md]: Thanks! fixed in 2a58580dec9

File: docs/contributing.md (hunk 2 above)
Comment (kate-goldenring) [reply docs/contributing.md]: Thanks fixed in 2a58580dec9

**Pull request merged**